src/linear_algebra/bilinear_form.lean

```diff
@@ -501,6 +520,17 @@ rfl
 lemma matrix.to_bilin'_apply (M : matrix n n R₃) (x y : n → R₃) :
   matrix.to_bilin' M x y = ∑ i j, x i * M i j * y j := rfl
 
+lemma matrix.to_bilin'_apply' (M : matrix n n R₃) (v w : n → R₃) :
+  (matrix.to_bilin' M) v w = matrix.dot_product v (M.mul_vec w) :=
```

Comment:
```suggestion
  M.to_bilin' v w = matrix.dot_product v (M.mul_vec w) :=
```
does this work?  If so, same in the lemma above it.

Reply:
I get the error `invalid field notation, function 'matrix.to_bilin'' does not have explicit argument with type (matrix ...)`, probably because it is a linear equiv